docker/Dockerfile.ngen

```diff
@@ -19,36 +18,19 @@ ARG REPO_URL \
     WORKDIR \
     UPDATE=2023-04-27
 
-WORKDIR ${WORKDIR}
+WORKDIR /ngen
 
-RUN cd ${WORKDIR} \
-    && if [ "x$COMMIT" != "x" ]; then \
-        git clone --single-branch --branch $BRANCH $REPO_URL \
-        && cd ./ngen \
-        && git checkout $COMMIT; \
+RUN if [ "x$COMMIT" != "x" ]; then \
+    git clone --single-branch --branch $BRANCH $REPO_URL \
+    && cd ./ngen \
+    && git checkout $COMMIT; \
     else \
-        git clone --depth 1 --branch $BRANCH $REPO_URL \
-        && cd ./ngen ; \
+    git clone --depth 1 --branch $BRANCH $REPO_URL \
+    && cd ./ngen ; \
     fi \
-    && echo "#!/bin/bash" > build_sub \
-    && echo "cmake -B \$1/cmake_build -DCMAKE_BUILD_TYPE=${NGEN_BUILD_CONFIG_TYPE} -S \$1" >> build_sub \
-    && echo "cmake --build \$1/cmake_build" >> build_sub \
-    #&& echo "cd \$1/cmake_build && make install" >> build_sub \
-    && chmod u+x build_sub \
     && git submodule update --init --depth 1 test/googletest \
     && git submodule update --init --recursive --depth 1
 
-################################################################################################################
-################################################################################################################
-FROM awiciroh/ngen-deps:${TAG_NAME} as rocky_ngen_build_testing
```

Comment:
These lines also need to recovered. We need all four image built with the same tag name.